cloud/test/test/testLogFile.txt

```diff
@@ -0,0 +1,709 @@
+1/7 09:15:19.344  ZONE_CHANGE,1825,"Hook Point",0
```

Comment:
the path "test/test/testLogFile" - might be better to make it "test/files/testLogFile"?